tree/dataframe/test/dataframe_concurrency.cxx

```diff
@@ -76,4 +77,72 @@ TEST(RDFSimpleTests, ThrowOnPoolSizeMismatch)
       }
    }
 }
+
+
+void SimpleParallelRDFs()
+{
+   // Run the RDF construction and the event loop in parallel
+   auto func = [] {
+         ROOT::RDataFrame df(10);
```

Comment:
nice to have to stir things up, but feel free to disregard: a sleep for a small, random amount of time, around 10% of the total execution time